modules/blocks/api.js

```diff
@@ -106,6 +106,16 @@ __private.list = function(filter, cb) {
 		params.height = filter.height;
 	}
 
+	if (filter.fromTimestamp >= 0) {
```

Comment:
In swagger we say `0` is the valid value. Should not we simplify it on API level to disallow zero as param value? 

https://github.com/LiskHQ/lisk/blob/5acfc7bdd902965124263d448be124edf481b6ae/schema/swagger.yml#L1169-L1174

Reply:
The idea is to keep `0` as an allowed value. However, there was a mistake in the expectation of `toTimestamp` that I've caught thanks to this comment. @nazarhussain could you check it again? I hope now it's easier to understand.